tests/gordo_components/client/test_client.py

```diff
@@ -490,3 +493,29 @@ def test_client_endpoint_filtering(
         assert (
             expected == filtered_endpoints
         ), f"Not equal: {expected} \n----\n {filtered_endpoints}"
+
+
+def test_exponential_sleep_time(watchman_service):
+    endpoint = _endpoint_metadata("t1", False)
+
+    start, end = (
+        isoparse("2016-01-01T00:00:00+00:00"),
+        isoparse("2016-01-01T12:00:00+00:00"),
+    )
+
+    with patch(
+        "gordo_components.client.client.time.sleep", return_value=None
+    ) as time_sleep:
+        client = Client(project=tu.GORDO_PROJECT)
+        x = pd.DataFrame([123])
+        y = None
+        loop = asyncio.get_event_loop()
+        loop.run_until_complete(
+            client._process_post_prediction_task(
+                X=x, y=y, chunk=slice(0, 1), endpoint=endpoint, start=start, end=end
+            )
+        )
+        loop.close()
+
+    calls = [call(8), call(16), call(32), call(64), call(128)]
```

Comment:
maybe call this variable something slightly more expressive, like "expected_calls" or something?